### ISSUE TITLE
pypi package

### ISSUE THREAD

**Issue by kvarga** [gpailler/AtlassianBot#7]
Instead of this all being basically a repository of a `slackbot` instance, have you thought about separating the modules into individual package?  ( I've done this at https://github.com/kvarga/slackbotjira )

**Comment by gpailler** [comment]
No, I didn't yet. My current usage is to activate and use all plugins but I understand people maybe want only one of them. Can be a good enhancement
